routes/web.php

```diff
@@ -11,6 +11,6 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', 'AdminController@index');
+Route::resource('/product', 'ProductController');
+Route::resource('/brand', 'BrandController');
```

Comment:
```suggestion
Route::resource('/products', 'ProductController');
Route::resource('/brands', 'BrandController');
```
viết ntn cho đúng chuẩn nhé